modules/launcher/src/main/java/org/mule/module/launcher/MuleDeploymentService.java

```diff
@@ -104,6 +106,7 @@ public void start()
             try
             {
                 listener.onAfterStartup();
+                this.isStarted = true;
```

Comment:
Shouldn't this be set before the listeners? I mean, I would expect the isStarted to be true if I'm executing onAfterStartup() method.